charts/reporting-operator/templates/custom-resources/prom-queries/pvc-capacity.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportPrometheusQuery
+metadata:
+  name: pvc-capacity-bytes
+spec:
+  query: |
+    max(kubelet_volume_stats_capacity_bytes) by (exported_namespace, persistentvolumeclaim) 
```

Comment:
We don't actually want the max. We want the value at every moment in time, and we want all the metric labels, so this can just be `kubelet_volume_stats_capacity_bytes`.